src/main/java/it/gov/pagopa/pu/workflow/service/debtposition/custom/fine/DebtPositionFineService.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.workflow.service.debtposition.custom.fine;
+
+import it.gov.pagopa.payhub.activities.dto.debtposition.syncwfconfig.FineWfExecutionConfig;
+import it.gov.pagopa.pu.workflow.dto.PaymentEventRequestDTO;
+import it.gov.pagopa.pu.workflow.dto.generated.WorkflowCreatedDTO;
+
+public interface DebtPositionFineService {
+
+  WorkflowCreatedDTO handleFineReductionExpiration(Long debtPositionId, PaymentEventRequestDTO paymentEventRequestDTO, boolean massive, FineWfExecutionConfig executionParams, String accessToken);
```

Comment:
I see you are asking the accessToken in order to give it as input for a WF:
* A WF cannot use accessToken provided as input! they could expire! a WF is supposed to auto-generate it when needed
```suggestion
  WorkflowCreatedDTO expireFineReduction(Long debtPositionId);
```